backend/src/main/java/com/soen341/instagram/security/UserDetailsImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.soen341.instagram.security;
+
+import java.util.Collection;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import com.soen341.instagram.model.Account;
+
+public class UserDetailsImpl implements UserDetails
```

Comment:
I'm not too sure how dead set we are on writing everything down with no abbreviations, but for class names I think we could make it a valid point to not shorten anything just to make sure it's extra clear what this class does from a functional point of view: so this would become `UserDetailsImplementation`

Reply:
In my opinion, I think it is fine to keep it as `impl`. As for your point on writing abbreviations, I think it is fine as long as the abbreviation is clear enough and the variable name is self explanatory.